docs/community/third-party-packages.md

```diff
@@ -153,6 +153,7 @@ To submit new content, [create a pull request][drf-create-pr].
 * [drf-viewset-profiler][drf-viewset-profiler] - Lib to profile all methods from a viewset line by line.
 * [djangorestframework-features][djangorestframework-features] - Advanced schema generation and more based on named features.
 * [django-elasticsearch-dsl-drf][django-elasticsearch-dsl-drf] - Integrate Elasticsearch DSL with Django REST framework. Package provides views, serializers, filter backends, pagination and other handy add-ons.
+* [django-lisan](https://github.com/Nabute/django-lisan) - A lightweight translation and localization framework for Django REST Framework APIs. PyPI: [django-lisan](https://pypi.org/project/lisan)
```

Comment:
Actually, could you follow the existing convention and not put the links inline?